api/dispatcher.proto

```diff
@@ -126,6 +126,9 @@ message SessionMessage {
 	// Symmetric encryption key distributed by the lead manager. Used by agents
 	// for securing network bootstrapping and communication.
 	repeated EncryptionKey network_bootstrap_keys = 4;
+
+	// Which root certificates to trust
+	bytes RootCA = 5;
```

Comment:
Why does the SessionMessage need a new RootCA field? Don't the nodes know exactly who to trust from the RootCA inside of the message that comes from the renewal of the certificate?